modules/qna/src/views/full/Components/QnA.tsx

```diff
@@ -1,14 +1,16 @@
 import { Button, Icon, Position, Tooltip } from '@blueprintjs/core'
 import { Flow, FlowNode } from 'botpress/sdk'
 import { confirmDialog, lang, MoreOptions, MoreOptionsItems } from 'botpress/shared'
-import { getFlowLabel } from 'botpress/utils'
+import { getFlowLabel, toastInfo } from 'botpress/utils'
```

Comment:
Please use `toast.info` from botpress/shared instead, we are getting rid of 'toastInfo' in dev (will save a conflict later)

Reply:
@allardy not sure if toast.info exists on master yet so it might be the only option